com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/UV/RadialShearNode.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class RadialShearNode : IStandardNode
+    {
+        public static string Name = "RadialShear";
+        public static int Version = 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"
+{
+    delta = UV - Center;
+    delta2 = dot(delta.xy, delta.xy);
+    delta_offset = delta2 * Strength;
+    tmep.x = delta.y;
```

Comment:
This variable is named incorrectly, should be `temp`, causes a shader error currently: 
![image](https://user-images.githubusercontent.com/13033542/167045221-8893701b-7003-4fdf-a4c8-756657183739.png)

Reply:
omg a terrible typo, thank you for catching that